tests/test_integration.py

```diff
@@ -98,6 +98,35 @@ def tar_from_directory(directory):
     return fileobj
 
 
+def container(name, fileobj):
```

Comment:
This seems like it copy/pastes `image()`. Presumably there should be a way for you to call `image()` instead, though I'm not 100% sure what the syntax is given we're dynamically generating those fixtures. But yeah have a look quickly at least?

Reply:
Yeah actually I tried doing this only but the value which image() is returning is a function which is leading into an error while creating the json containing data for container to get started.

I'll look into it further and see if there is a way in which this can be done.